src/dlgConnectionProfiles.h

```diff
@@ -138,6 +138,7 @@ public slots:
     QPushButton* connect_button;
     QLineEdit* delete_profile_lineedit;
     QPushButton* delete_button;
+    QAction* copyProfile;
```

Comment:
I'd welcome marking this as a member with a prefix - it does help if one is looking at the code outside of an IDE that automagically works out that it *is* a member, e.g. in a git diff. It helps to avoid leaving a local instance of the variable shadowing the intended member - as would happen if one had left the `auto` in the constructor where it was previously created as a local.
```suggestion
    QAction* mpCopyProfile;
```

Reply:
Applied